helpers/sherpa_config.py

```diff
@@ -85,6 +88,12 @@ def finalize_options(self):
         if self.region_lib_dirs is None:
             self.region_lib_dirs=self.install_dir+'/lib'
 
+        # It is not clear that the other boolean options work
+        # correctly when set to False/false.
+        #
+        if not isinstance(self.region_use_cxc_parser, bool):
+            self.region_use_cxc_parser = self.region_use_cxc_parser.lower() == "true"
+
```

Comment:
I don't really understand how the config value from `setup.cfg` get converted to Python values. I would imagine it just uses string conversion, but some of our options seem to treat `"true"` / `"false"` values as booleans (or maybe all they really care about is if the value is set or not, the value is irrelevant), which makes me wonder just what is going on,

Reply:
By itself, all values are read as strings, put configparser has "getboolean", "getint" and "getfloat" methods that retrieve the value and perform conversion at that same time: https://docs.python.org/3/library/configparser.html#supported-datatypes